manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -69,48 +85,247 @@ public void init(Container container) throws Exception {
                         container.getService(TimerService.class),
                         identityService, this)
         );
+
+        Predicate<Path> checkFilePred = (path) -> Files.isRegularFile(path.toAbsolutePath());
+
+        Path defaultMapSettingsPath = Stream.of("/opt/map/mapsettings.json", "manager/src/map/mapsettings.json")
+                .map(Path::of)
+                .filter(checkFilePred)
+                .toList().get(0);
+
+        Path defaultMapTilesPath = Stream.of("/deployment/map/mapdata.mbtiles", "/opt/map/mapdata.mbtiles", "manager/src/map/mapdata.mbtiles")
+                .map(Path::of)
+                .filter(checkFilePred)
+                .toList().get(0);
+
+        mapTilesPath = Paths.get(getString(container.getConfig(), OR_MAP_TILES_PATH, defaultMapTilesPath.toAbsolutePath().toString()));
+        mapTilesPath = Files.isRegularFile(mapTilesPath) ? mapTilesPath : defaultMapTilesPath;
+
+        mapSettingsPath = Paths.get(getString(container.getConfig(), OR_MAP_SETTINGS_PATH, defaultMapSettingsPath.toAbsolutePath().toString()));
+        Path persistencePath = persistenceService.getStorageDir().resolve("manager").resolve("mapsettings.json").toAbsolutePath();
+
+        if(!Files.isRegularFile(mapSettingsPath)) {
+            if(!Files.isRegularFile(persistencePath)){
+                persistencePath.getParent().toFile().mkdirs();
+                LOG.warning("Copying defaults to and using " + persistencePath.toString() + " for mapsettings.json");
+                Files.copy(defaultMapSettingsPath, persistencePath);
+                mapSettingsPath = persistencePath;
+            }
+        }
+        if(Files.isRegularFile(persistencePath)) {
+            mapSettingsPath = persistencePath;
+        }
+        LOG.info("Configuration Service Used files:");
+        LOG.info("\t- manager_config.json: " + getManagerConfigPath().toAbsolutePath().toString());
+        LOG.info("\t- mapsettings.json: " + mapSettingsPath.toAbsolutePath().toString());
+        LOG.info("\t- mapdata.mbtiles: " + mapTilesPath.toAbsolutePath().toString());
```

Comment:
Redundant `toString` calls:

```suggestion
        LOG.info("\t- manager_config.json: " + getManagerConfigPath().toAbsolutePath());
        LOG.info("\t- mapsettings.json: " + mapSettingsPath.toAbsolutePath());
        LOG.info("\t- mapdata.mbtiles: " + mapTilesPath.toAbsolutePath());
```